x-pack/plugins/serverless_observability/public/plugin.ts

```diff
@@ -30,6 +30,7 @@ export class ServerlessObservabilityPlugin
   ): ServerlessObservabilityPluginStart {
     const { observabilityShared, serverless } = setupDeps;
     observabilityShared.setIsSidebarEnabled(false);
+    serverless.setProjectHome('/app/enterprise_search/content/setup_guide');
```

Comment:
Shouldn't be `/app/observability/landing ` ?
https://github.com/elastic/kibana/blob/ec7ba022e19ec3a616bd6ed9a3065bff7402c02a/config/serverless.oblt.yml#L15

Reply:
Should be, it was already there in `<DefaultNavigation`

Reply:
Thank you! I corrected this in 2fba2ac2a75.

Now that you bring up the fact that this is already in `uiSettings`, I'm wondering why the home page couldn't just be read from there when it is needed internally in the project header.

I'll look into this more as it could be an easy solution to just use UI Settings.